### PR TITLE
Guruc-134 markdown previewer

### PR DESCRIPTION
**Pull request by guruc-134**
## Related Issue
  New Project: Markdown Previewer
  
Closes: #[issue number that will be closed through this PR]
issue: #299 
### Describe the changes you've made
This is a markdown previewer, Here the user enters his text in the markdown format, and it gets rendered dynamically and  displayed on the previewer

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
`NA`

### Checklist:

- [ x ] My code follows the style guidelines of this project.
- [ x ] I have performed a self-review of my own code.
- [ x ] I have commented my code, particularly in hard-to-understand areas.
- [ x ] I have made corresponding changes to the documentation.
- [ x ] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.

image

### CODE REVIEW

**Comment by urvashi-code1255** [comment]
@guruc-134 Resolve the mergee conflicts

**Comment by guruc-134** [comment]
@urvashi-code1255  removed the duplicate file names present in the scripts.js

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ridsuteri) [inline .vscode/settings.json:1]: any use of this file?

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@guruc-134 add this project to the script.js file

**Pull request closed without merging**